packages/wallet/src/redux/state.ts

```diff
@@ -67,6 +68,15 @@ export interface ProcessState {
   channelsToMonitor: string[];
 }
 
+export interface FundingState {
```

Comment:
I think we'll need some permanent store of whether a channel is funded directly or indirectly and that stores associations between ledger channels and the channels they fund. I'm not sure if this should be in `ChannelState` or if we should have a `FundingState`.